### PR TITLE
Wildfire QA round 2 updates

### PR DESCRIPTION
**Pull request by clhenrick**
- [x] Fix month in chart view title
- [x] limit wildfire probability value labels to one significant digits 
- [x] fix y-axis min / max values in Chart 
- [x] use mean aggregation for wildfire probability 
- [ ] ~~display percent of no-data for boundaries~~

### CODE REVIEW

**Comment by clhenrick** [comment]
[Tools: Widlfire](https://trello.com/c/i33A7gwb/49-tools-widlfire)

File: src/routes/tools/wildfire/_data.js [hunk 1]
```diff
@@ -142,25 +142,27 @@ export function getQueryParams({
...
     imperial,
+    stat: climvar === "fire" ? "sum" : "mean",
+    // countnd: true,
     ...(simulation === "month" && { months: monthNumber }),
   };
   let method;
   switch (boundary.id) {
     case "locagrid":
       params.g = `Point(${location.center[0]} ${location.center[1]})`;
+      params.stat = "mean";
```
Comment (mukhtyar) [inline src/routes/tools/wildfire/_data.js]: you dont need the `stat` param for the locagrid. although it doesn't look like it changes the values.

**Review by mukhtyar — APPROVED** [review]

File: src/routes/tools/wildfire/_data.js (hunk 1 above)
Comment (clhenrick) [reply src/routes/tools/wildfire/_data.js]: Got it, I guess I could delete it here instead

**Comment by clhenrick** [comment]
Going to merge this and address the last task (display percent of no-data for boundaries) in a separate PR.

**Pull request merged**